.github/CONTRIBUTING.md

```diff
@@ -1,30 +1,38 @@
-###Thank you for taking the time to contribute to the WildBeast documentation!
+# Thank you for taking the time to contribute to the WildBeast documentation
 
 Before you start making the changes to the documentation, let's clarify the following. **If you make changes, you must know what you're doing with WildBeast and/or the operating system in question.**
-  
+
 Any larger additions or restructurations must be discussed with the WildBeast developers!
 
-#Rules
+----
+
+## Rules
+
 Please follow these rules when making contributions to this repository.
 
 ## General contribution rules
+
 1. All pull requests go to the review branch.
-2. You must have good written English skills, pulls with grammar errors will get change requests.
-3. Follow the overall format in the docs.
-4. Leave image embedding to the official writers.
+1. You must have good written English skills, pulls with grammar errors will get change requests.
```

Comment:
Done in [d8ecdec](https://github.com/TheSharks/WildBeastDocs/pull/20/commits/d8ecdecfe1bdba998bb04c9c2d8a9566c1399b8c)